src/client/styles/components/_header.scss

```diff
@@ -1,29 +1,30 @@
 .header {
-  padding: 40px 16px 20px;
-  background: linear-gradient(rgba(#fff, 0.1), rgba(#fff, 0.05));
+  background: linear-gradient(rgba($white, 0.1), rgba($white, 0.05));
   margin-bottom: 8px;
-}
+  padding: 40px 16px 20px;
 
-.header__title {
-  overflow: hidden;
-  text-overflow: ellipsis;
-  margin-right: 8px;
-  white-space: nowrap;
-  flex-grow: 1;
-}
+  &__title {
```

Comment:
I'd rather not nest bem elements like this. I've never actually used `stylelint` (shame on me ☹️) so I'm struggling to configure the rules so it won't yell at me for reverting to `header__title`, can you either fix it or let me know how? I'd have thought the bem plugin would allow that but it doesn't seem to.

Reply:
Oh, the files are SCSS. Did you not want to use Sass?

Reply:
I think I wasn't totally clear, it's specifically the nesting of BEM elements I'd like to avoid.